application/src/main/java/org/opentripplanner/updater/vehicle_rental/datasources/params/AllowedRentalType.java

```diff
@@ -0,0 +1,7 @@
+package org.opentripplanner.updater.vehicle_rental.datasources.params;
+
+public enum AllowedRentalType {
```

Comment:
Please implement `DocumentedEnum` which makes it trivial to keep documentation up to date. You can use it like this: https://github.com/opentripplanner/OpenTripPlanner/blob/f4bfedfc41d3b92df45927a72edcab28624510d6/application/src/main/java/org/opentripplanner/standalone/config/routerequest/ItineraryFiltersConfig.java#L70-L77